<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dynamics_page</class>
 <widget class="QWidget" name="dynamics_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="TransparentTogglePushButton" name="dm_info_button">
       <property name="text">
        <string>Dynamic Model Infomation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TransparentTogglePushButton" name="phase_plane_analysis_button">
       <property name="text">
        <string>Phase Plane Analysis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TransparentTogglePushButton" name="bifurcation_analysis_button">
       <property name="text">
        <string>Bifurcation Analysis</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="dm_info_page">
      <layout class="QVBoxLayout" name="dm_info_page_verticalLayout"/>
     </widget>
     <widget class="QWidget" name="phase_plane_page">
      <layout class="QVBoxLayout" name="phase_plane_page_verticalLayout"/>
     </widget>
     <widget class="QWidget" name="bifurcation_page">
      <layout class="QVBoxLayout" name="bifurcation_page_verticalLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToggleButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TransparentTogglePushButton</class>
   <extends>ToggleButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
